### ISSUE TITLE
Steam Washer (Ore Washer Mode) - Steam Usage Too High (Balance)

### ISSUE THREAD

**Issue by Dyonovan** [GTNewHorizons/GT-New-Horizons-Modpack#17223]
### Your GTNH Discord Username

Dyonovan

### Your Pack Version

Nightly-612

### Your Proposal

Compared to other steam multi's, the amount of steam the Steam Washer uses seems high. I have 2-High Pressure Solar Boilers, 3-Simple Boilers, and 4-High Pressure Coal Boilers and this does not keep up with the Steam Washer.

Was washing 8 Crushed Lead Ore and it uses 171 L/tick (3420 L/Second)

### Your Goal

Balance the steam usage to compare to other Steam Multi's. I really don't have a suggestion to what.

### Your Vision

Steam usage between Steam Multi's should be comparable for their jobs. 

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by chochem** [comment]
is that in line with the tooltip or are you saying its bugged?

**Comment by Dyonovan** [comment]
Its in line with the tooltip. Its working fine.

**Comment by NotAPenguin0** [comment]
3420L/s is only 2A LV, I think that's totally fair for full ore washing

**Comment by chochem** [comment]
> Its in line with the tooltip. Its working fine.

then I dont know what you are saying. As far as I know all the steam multis have the same discounts

**Comment by Dyonovan** [comment]
Then maybe the other Steam Multi's should be increased. All I am saying is compared to all other multi's, which I can run multiple with my current steam product, yet I can't just run the washer without it eating all my steam.

Just a balance suggest

> 3420L/s is only 2A LV, I think that's totally fair for full ore washing. Yes, you probably can't feed this in steam age/early LV, but with the power you get from this multi it can easily scale well into MV when supplying this amount is no problem at all.

**Comment by chochem** [comment]
are you just saying that washing is more power intense than macerating?

**Comment by dvdmandt** [comment]
The ore washer LV machine is also significantly more energy intensive (16 EU/t) than macerator or centrifuge for ore processing (2 EU/t and 5 EU/t), and the compressor typically uses 2 EU/t as well.

**Comment by Dyonovan** [comment]
Yes. I'm not necessarily saying it should be decreased, im just saying compare to other multi's it doesn't feel right.

> are you just saying that washing is more power intense than macerating?
